### ISSUE TITLE
Set headerColor programatically

### ISSUE THREAD

**Issue by josephchua07** [Applandeo/Material-Calendar-View#65]
Hi does anybody know a way to set headerColor programically so it will change the color depending on the theme that user wants.

`<com.applandeo.materialcalendarview.CalendarView
                    android:id="@+id/material_calendar"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    app:headerColor="@color/material_red_700"
                    />`

**Comment by kormateusz** [comment]
Hi, I don't know 😂 Currently it's impossible but.... you can download our code and implement such functionality yourself ;)

**Comment by josephchua07** [comment]
Hi! Thanks for your response :)
I actually tried your suggestion to implement such function.. but it seems that there is no need to do that anymore. I found the answer under the CalendarView initAttributes(). We can just use the AppearanceUtils to set colors programatically :)


in CalendarView initAttributes()
`        private void initAttributes() {
        AppearanceUtils.setHeaderColor(getRootView(), mCalendarProperties.getHeaderColor());
        AppearanceUtils.setHeaderLabelColor(getRootView(), mCalendarProperties.getHeaderLabelColor());
        ........
    }
`


In my Activity
`AppearanceUtils.setHeaderColor(material_calendar, getContext().getResources().getColor(R.color.resPrimary));`
